roles/openshift_aws/tasks/dns_route53_record.yml

```diff
@@ -0,0 +1,25 @@
+---
+- debug: msg="{{ l_openshift_dns_records }}"
+
+- name: querying elb
+  ec2_elb_facts:
+    names: "{{ l_openshift_dns_records['value'] }}"
+    region: "{{ openshift_aws_region }}"
+  register: elb_facts
+  when:
+    - "'elb' in l_openshift_dns_records['kind']"
+    - "'CNAME' in l_openshift_dns_records['type']"
+
+- name: creating record
+  route53:
+    command: create
+    overwrite: no
+    private_zone: "{{ (l_openshift_dns_records['scheme'] == 'internal') | ternary('yes','no') }}"
+    record: "{{ l_openshift_dns_records['name'] }}.{{ l_openshift_dns_records['zone'] }}"
+    type: "{{ l_openshift_dns_records['type'] }}"
+    ttl: 300
+    value: "{{ value | trim }}"
+    zone: "{{ l_openshift_dns_records['zone'] }}"
+  vars:
+    value: "{% if 'CNAME' in l_openshift_dns_records['type'] -%} {{ elb_facts.elbs[0].dns_name }} \
```

Comment:
If it's not a CNAME?  I'm sure it's supposed to get a value either way.  This seems unnecessary.